java/src/org/certificatetransparency/ctlog/MerkleTreeLeaf.java

```diff
@@ -0,0 +1,16 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class MerkleTreeLeaf {
```

Comment:
If you declare them as public final they can only be set once at construction. In this case though the objects are presumably modifiable so it's unclear as to whether this is a great help.